### ISSUE TITLE
Document access to Web-Admin via HTTPS

### ISSUE THREAD

**Issue by Cito** [zatosource/zato-docs#26]
On the page https://zato.io/docs/web-admin/security.html I was missing instructions on making the Web-Admin accessible via HTTPS only - actually I think that should be the default. The login password and other passwords you can set with the Web-Admin should not go unencrypted over the line.

**Comment by Cito** [comment]
I've got it working now by using nginx as a reverse proxy, forwarding https://zatoserver to http://localhost:8183. However, I found it was no so easy because Zato uses an old Django 1.3.7 which does not support Django settings like `SECURE_PROXY_SSL_HEADER`. Therefore, Django is not able to understand it is serving SSL in this configuration. As a workaround, I set `os.environ['HTTPS'] = 'on'` in the main.py file of the Web-Admin. But I guess this is a good reason to update Django. Other security related settings such as `SESSION_COOKIE_SECURE` are not supported by Django 1.3.7 either.
